### ISSUE TITLE
Get rid of the Test Pilot Promo on experiment details pages

### ISSUE THREAD

**Issue by johngruen** [mozilla/testpilot#3507]
See #3504 and #3503 
If we move experiment controls to the sidebar we can move the one click install component there as well. This removes the need for the Test Pilot Promo header for users w/o test pilot installed.

**Comment by johngruen** [comment]
As in, nuke this:

<img width="1299" alt="screen shot 2018-04-16 at 3 56 16 pm" src="https://user-images.githubusercontent.com/3323249/38831874-c041d47e-418e-11e8-87d9-56808c001297.png">

**Comment by dannycoates** [comment]
@sevaan could you spec up what the one-click install should look like with the side controls from #3503? I tried just putting the existing green button there but it looks horrible. #3504 says we'll also need the legal text also. Thanks :)

I assume the button will need to handle all the special cases as well, like non-firefox and outdated.

**Comment by dannycoates** [comment]
I suggest we keep the promo for non-firefox and outdated firefox case while one-click should move to the sidebar.

**Comment by sevaan** [comment]
Button plus legal in the sidebar:

<img width="341" alt="image" src="https://user-images.githubusercontent.com/473531/39444348-659d8bd8-4c85-11e8-8080-10ccb7925750.png">

I removed the border of the button, and matched the width of the other buttons in there. Legal text has been moved to that grey area and is 11px in size, #000000

> I suggest we keep the promo for non-firefox and outdated firefox case while one-click should move to the sidebar.

I agree. The "Download Firefox" button doesn't make much sense in that sidebar. I think the current header is better and separates downloading Fx from the context of the experiment.
